tests/unit/script/htmlimageelement.rs

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use script::test::DOMString;
+use script::test::sizes::{parse_a_sizes_attribute, Size, test_media_query, test_length};
+
+#[test]
+fn no_default_provided() {
+    assert!(parse_a_sizes_attribute(DOMString::new(), None).len() == 1);
```

Comment:
Instead of asserting the length, we should be asserting the actual value that is returned in all of these tests.